src/apps/common/submodules/portWizard/portWizard.scss

```diff
@@ -1,4 +1,5 @@
 @import '../../../../css/partials/base';
+@import 'partials/_step-nameAndNumbers.scss';
```

Comment:
This import should be nested inside the common control root selector to avoid the possibility of targeting unwanted elements (global pollution).